mobile.css

```diff
@@ -57,6 +58,38 @@ body {
     order: 0;
     flex-grow: 0;
 }
+.my-name{
+    font-family: 'Crete Round';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 56px;
+    line-height: 64px;
+    color: #FF6B00;
+    order: 0;
+}
+.supporting-text{
+    font-family: 'Roboto', 'san-serif';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+    display: flex;
+    align-items: center;
+    color: #FFF5E1;
+    order: 1;
+}
+.description{
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    order: 2;
+    color: #FFFFFF;
+}
+#item-navigation{
+    display: none;
+}
```

Comment:
[OPTIONAL] There are so many disparities between your design and that of the figma in the mobile version which does not give your website a nice appearance and hinders it from becoming responsive as shown below. For instance, there is a black space to the right of the page that runs from the headline section downwards. Kindly take your time and through the whole website and fix these anomalies. 

![Screenshot from 2023-01-27 17-56-25](https://user-images.githubusercontent.com/65233860/215167172-52fc753b-fb61-4b45-b8bd-c3e7506d90f6.png)

![Screenshot from 2023-01-27 17-55-58](https://user-images.githubusercontent.com/65233860/215167064-45eb1757-3d2f-49a7-b30e-83ba5b7e3654.png)

![Screenshot from 2023-01-27 17-56-11](https://user-images.githubusercontent.com/65233860/215167087-8caf0669-1ed2-4c69-aa78-9bb582f6afea.png)


![Screenshot from 2023-01-27 17-55-41](https://user-images.githubusercontent.com/65233860/215167264-abef7e21-5368-434a-995b-21c015336bab.png)